CMakeLists.txt

```
@@ -30,7 +30,8 @@ set(ignoreMe "${VLLM_PYTHON_PATH}")
 set(PYTHON_SUPPORTED_VERSIONS "3.9" "3.10" "3.11" "3.12")
 
 # Supported NVIDIA architectures.
-if(${CMAKE_CUDA_COMPILER_VERSION} VERSION_GREATER_EQUAL)
+if(DEFINED CMAKE_CUDA_COMPILER_VERSION AND
+   CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL "12.0")
```

Doesn't this need to be 12.8?
```suggestion
   CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL "12.8")
```